scripts/devcloud-test.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+# Copyright (C) 2023 Intel Corporation
+
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+workspace=$1
+compiler_c=$2
+compiler_cxx=$3
+build_type=$4
+
+set -e
+hostname
+
+# SLURM/MPI integration is broken
+unset SLURM_TASKS_PER_NODE
+unset SLURM_JOBID
+
+source /opt/intel/oneapi/setvars.sh
+source ${workspace}/dpcpp_compiler/startup.sh
+export PATH=${workspace}/dpcpp_compiler/bin:$PATH
+export CPATH=${workspace}/dpcpp_compiler/include:$CPATH
+export LIBRARY_PATH=${workspace}/dpcpp_compiler/lib:$LIBRARY_PATH
+export LD_LIBRARY_PATH=${workspace}/dpcpp_compiler/lib:$LD_LIBRARY_PATH
+
+cmake \
+-B${workspace}/build \
+-DCMAKE_C_COMPILER=${compiler_c} \
+-DCMAKE_CXX_COMPILER=${compiler_cxx} \
+-DUR_ENABLE_TRACING=ON \
+-DUR_DEVELOPER_MODE=ON \
+-DCMAKE_BUILD_TYPE=${build_type} \
+-DUR_BUILD_TESTS=ON \
+-DUR_FORMAT_CPP_STYLE=OFF \
+-DUR_BUILD_ADAPTER_L0=ON
+
+cmake --build ${workspace}/build -j $(nproc)
+
+unset LD_LIBRARY_PATH
+unset LIBRARY_PATH
```

Comment:
Why do you unset these?

Reply:
It seems that `startup.sh` prepends some paths instead of overwriting them, so when the `exports` are removed, there will be no need for unsettling those variables.

Reply:
I tried these tests without unsetting these variables and I had a lookup error.
there is no startup.sh script in the new dpc++ package. In my exports, I prepend some paths in the same way.

Reply:
If these two exports are needed for building only, it would be cleaner if you just prepended the build command with required `LD_LIBRARY_PATH` and `LIBRARY_PATH`.

Reply:
I have changed it